src/frequenz/client/dispatch/_client.py

```diff
@@ -0,0 +1,101 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Dispatch API client for Python."""
+from dataclasses import dataclass
+from datetime import datetime
+
+import grpc
+from frequenz.api.dispatch.v1 import dispatch_pb2, dispatch_pb2_grpc
+from frequenz.sdk.microgrid.component import ComponentCategory
```

Comment:
This should come from https://github.com/frequenz-floss/frequenz-client-common-python/. We still need to move/recreate the wrappers in that repo (they are not available yet).

@frequenz-floss/api-reporting-team @frequenz-floss/api-microgrid-team FYI as this is common to at least all these APIs